bin/images-imgadm

```diff
@@ -23,9 +23,9 @@ var common = require('../lib/common');
 //---- globals
 
 var NAME = 'joyent-imgadm';
-var URL = process.env.JOYENT_IMGADM_URL || 'https://images.joyent.com';
+var URL = process.env.JOYENT_IMGADM_URL || 'https://images.smartos.org';
```

Comment:
Not a blocker, but how hard would it be to s/JOYENT/SMARTOS/g for the variable?  It might stretch its tentacles too far to be useful to fix right now.